docs/mswms.rst

```diff
@@ -507,12 +507,12 @@ At current state we have to use pip to install mod_wsgi into the INSTANCE enviro
 
 Setup a /etc/apache2/mods-available/wsgi_express.conf::
 
-  WSGIPythonHome "/home/mss-demo/miniforge/envs/demo/"
+  WSGIPythonHome "/home/mss-demo/miniforge/envs/mssenv/"
 
 
 Setup a /etc/apache2/mods-available/wsgi_express.load::
 
-  LoadModule wsgi_module "/usr/lib/apache2/modules/mod_wsgi-py37.cpython-37m-x86_64-linux-gnu.so"
+  LoadModule wsgi_module "/usr/lib/apache2/modules/mod_wsgi-py310.cpython-310m-x86_64-linux-gnu.so"
```

Comment:
Maybe add a comment to replace the python version with the used one? This is terrible to maintain...

Reply:
I think now it is better to understand how that works.